storage/storagecontext.py

```diff
@@ -0,0 +1,9 @@
+from util.ipresolver import NoopIPResolver
+
+
+class StorageContext(object):
```

Comment:
Why did this need to move?

Reply:
because it's being imported in `multicdnprovider.py` and `__init__.py` and  `multicdnprovider.py` is also being imported in `__init__.py` so I had to move it otherwise there will be an cyclic import